### ISSUE TITLE
Add Laravel 10 Support

### ISSUE THREAD

**Issue by caturandi-labs** [l3aro/pipeline-query-collection#16]
I have tried to install this package in Laravel 10 but got error. 

```
Problem 1
    - Root composer.json requires l3aro/pipeline-query-collection * -> satisfiable by l3aro/pipeline-query-collection[v0.1.0, ..., v0.1.4, v1.0.0, ..., v1.3.5].
    - l3aro/pipeline-query-collection[v0.1.0, ..., v0.1.4, v1.0.0, ..., v1.3.5] require illuminate/contracts ^9.0 -> found illuminate/contracts[v9.0.0, ..., v9.52.0] but these were not loaded, likely because it conflicts with another require.

You can also try re-running composer require with an explicit version constraint, e.g. "composer require l3aro/pipeline-query-collection:*" to figure out if any version is installable, or "composer require l3aro/pipeline-query-collection:^2.1" if you know which you need.
```

Can you fix? Thanks !

**Comment by l3aro** [comment]
Fixed in #15

**Comment by caturandi-labs** [comment]
SUPER !! Thanks mate
